.github/CONTRIBUTING.md

```diff
@@ -356,7 +356,7 @@ Once you sign a CLA, all your existing and future pull requests will have the st
 [testing-guidelines]: ../docs/testing-guidelines/testing-guidelines.md
```

Comment:
@bergmeister, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Common Tests - Validate Markdown Files.Should not have errors in any markdown files
```powershell
Expected strings to be the same, but they were different.
Expected length: 9
Actual length:   272
Strings differ at index 0.
Expected: '--EMPTY--'
But was:  '/home/vsts...'
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown.tests.ps1: line 100
100:         $markdownErrors -join "`n" | Should -BeExactly "--EMPTY--"
```

Reply:
@iSazonov I am not sure I understand this failure. Any suggestions?

Reply:
The test output is trimmed and useless. I see the same in #9864 and #9865. I hope @TravisEz13 helps us.

Reply:
You have to look at the actual tests logs.  The test needs to be rewritten to give good results.

Reply:
This is what is in the actual test log, it does not help more,,,
```
[-] Should not have errors in any markdown files 6.5s
      Expected strings to be the same, but they were different.
      Expected length: 9
      Actual length:   272
      Strings differ at index 0.
      Expected: '--EMPTY--'
      But was:  '/home/vsts...'
      100:         $markdownErrors -join "`n" | Should -BeExactly "--EMPTY--"
      at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown.tests.ps1: line 100
```

Reply:
> 
> 
> You have to look at the actual tests logs. The test needs to be rewritten to give good results.

Agreed.  `Write-Verbose` needs to be on the side of us contributors.

Reply:
In theory, you can install the markdownlint extension in VS Code.  I know this used to work. Perhaps it broke when we moved to AzDevOps

Reply:
I enhanced it to report the failure in Pester's `-Because` parameter (which is only evaluated when the test fails)